documentation/user-interface/test-editor.md

```diff
@@ -0,0 +1,70 @@
+
+The **Test** editor consists of the following parts:
+
+* The toolbar - Provides information about the test (the test name, fixture name, tested page's URL) and buttons for perfoming [common tasks](#common-tasks) .
+
+    You can click the fixture name to open the fixture in the [Fixture](#fixture-editor) editor. You can also click the webpage's URL to view the webpage in the browser.
+
+* The test area - Displays the current test's content.
+* The **Actions** panel - Lists [actions](../working-with-testcafe-studio/recording-tests/test-actions/README.md) you can add to the test.
+    The actions are organized into categories. You can expand or collapse the categories by clicking the arrows next to the categories' names.
+
+![Test editor](../../images/user-interface/test-editor.png)
+
+## Common Tasks
+
+You can perform the following tasks in the editor:
+
+* [Add Actions to Tests](#add-actions-to-tests)
+* [Record Actions](#record-actions)
+* [Run Tests](#run-tests)
+* [Modify Actions' Parameters](#modify-action-parameters)
+* [Change Actions' Position](#change-action-position)
+* [Rename Tests](#rename-tests)
+* [Delete Actions](#delete-actions)
+
+### Add Actions to Tests
+
+Click an action in the **Actions** panel to add the action to the test. TestCafe Studio adds the action to the end of the test and allows you to specify the action's parameters.
+
+IMAGE
+
+You can add actions after or during recording. In the last case, you can pick an element on a webpage to set it as an action parameter ([Element Selector](../working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/action-parameters.md#element-selector) or [Actual Parameter](../working-with-testcafe-studio/recording-tests/test-actions/assertions.md#actual-parameter)).
+
+### Record Actions
+
+You can record [on-page actions](../working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md) and append them to the current test.
+
+Click the ![Record button](../../images/working-with-testcafe-studio/record-test-icon.png) **Record test actions** button on the toolbar to start recording and wait while TestCafe Studio plays back the existing actions, then perform new actions on the tested page. TestCafe Studio records your activity and inserts new actions after the existing ones.
+
+### Run Tests
+
+Click the ![Run tests button](../../images/working-with-testcafe-studio/action-run-icon.png) **Run test** button on the editor's toolbar to run the test.
+
+### Modify Actions' Parameters
+
+Click the action's row to expand the action parameters area that allows you to view or modify the action's parameters.
+
+### Change an Action's Position
+
+Hover the mouse cursor over the ![Drag icon](../../images/user-interface/drag-item-icon.svg) button to the left of the action's name until the cursor icon changes to the ![Drag icon](../../images/user-interface/move-cursor-icon.png) icon, then click and drag the action to a new position.
```

Comment:
> then click and drag the action to a new position

click means that you press and release the mouse button. Here you don't need to release it, so you can just say `drag`.